rdf-star-cg-spec.html

```diff
@@ -558,22 +562,119 @@ <h2>Evaluation Semantics</h2>
+          </pre>
+          This term is represented in JSON as follows:
+          <pre>
+                {
+                  "type": "triple",
+                  "value": {
+                     "subject": {
+                        "type": "uri",
+                        "value" "http://example.org/alice"
+                     },
+                     "predicate": {
+                        "type": "uri",
+                        "value" "http://example.org/name"
+                     },
+                     "object": {
+                        "type": "literal",
+                        "value" "Alice",
+                        "datatype": "http://www.w3.org/2001/XMLSchema#string"
+                     },
+                  }
+                }
+          </pre>
+        </div>
+      <!-- <div class="issue" data-number="13"></div> -->
       </section>
 
       <section>
         <h2>SPARQL* Query Results XML Format</h2>
+       <p>
+        The result of a SPARQL SELECT query is serialized in XML as defined in [[[rdf-sparql-XMLres]]]. This format proposes an XML representation of variable bindings to RDF terms.
+       </p>
+       <p>To accomodate the new RDF term for <a>embedded</a> triples that RDF* introduces, the list of RDF terms and their XML representations in [<a href="rdf-sparql-XMLres#results">rdf-sparql-XMLres, Section 2.3.1</a>] is extended as follows:
+        </p>
+        <p>
+        <dl>
+            <dt>An <a>embedded</a> triple with subject term <em>S</em>, predicate term <em>P</em>, and object term <em>O</em></dt>
+          <dd>
+            <pre class="xml">
+             &lt;binding&gt;
+                &lt;triple&gt;
+                  &lt;subject&gt;<em>S</em>&lt;subject&gt;
+                  &lt;predicate&gt;<em>P</em>&lt;predicate&gt;
+                  &lt;object&gt;<em>O</em>&lt;object&gt;
+                &lt;/triple&gt;
+              &lt;/binding&gt;
+            </pre>
+          </dd>
+        </dl>
+        <div class="example">
+            Consider the following RDF term, an <a>embedded</a> triple in Turtle* syntax:
+          <pre data-transform="updateExample"
+            data-content-type="text/x-turtle-star"
+            class="nohighlight example"
+          >
+            <!--
+            << <http://example.org/alice> <http://example.org/name> "Alice" >>
+            -->
+          </pre>
+          This term is represented in XML as follows:
+          <pre class="xml">
+            &lt;triple&gt;
+                &lt;subject&gt;
+                    &lt;uri&gt;http://example.org/alice&lt;/uri&gt;
+                &lt;/subject&gt;
+                &lt;predicate&gt;
+                    &lt;uri&gt;http://example.org/name&lt;/uri&gt;
+                &lt;/predicate&gt;
+                &lt;object&gt;
+                    &lt;literal datatype='http://www.w3.org/2001/XMLSchema#string'&gt;Alice&lt;/literal&gt;
+                &lt;/object&gt;
+            &lt;/triple&gt;
+          </pre>
```

Comment:
```suggestion
          <pre data-transform="updateExample" class="xml example">
            <!--
            <triple>
                <subject>
                    <uri>http://example.org/alice</uri>
                </subject>
                <predicate>
                    <uri>http://example.org/name</uri>
                </predicate>
                <object>
                    <literal datatype='http://www.w3.org/2001/XMLSchema#string'>Alice</literal>
                </object>
            </triple>
            -->
          </pre>
```
as above, more readable.

Reply:
Ah, nice! I was looking for a way to do that but couldn't figure it out. So simple in the end :)